app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,18 @@
+class LikesController < ApplicationController
+  def create
+    @like = current_user.likes.new(like_params)
+    @post = Post.find(params[:like][:post_id])
+
+    if @like.save
+      redirect_to "#{users_path}/#{@post.author_id}/posts/#{@post.id}"
```

Comment:
- [ ] [OPTIONAL] Great job 👏 with this functionality. If you would like to make your code more succinct. you can use URL helpers that "Rails" provides to simplify navigation.
 ```suggestion
       redirect_to user_post_path(@post)
```